### PR TITLE
OCD 3434: Automation test for RWT upload 

### PR DESCRIPTION
**Pull request by palakpatel311**

### CODE REVIEW

File: e2e/resources/RWT_Upload_File.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿UNIQUE_CHPL_ID__C,TYPE,SUBMITTED,URL
+15.04.04.1206.SMP1.10.00.1.180608,Plan,20210216,https://www.myhealtitsoftware.com/rwt/myproduct1/plan/
+15.04.04.1447.SMP2.18.07.1.180927,Plan,20200223,https://www.myhealtitsoftware.com/rwt/myproduct2/plan/
+15.04.04.2902.SMP5.91.00.1.171016,Plan,,
```
Comment (andlar) [inline e2e/resources/RWT_Upload_File.csv:4]: I know these lines (the ones without data) should trigger an error, but the tests don't test that. Not sure if we should have "bad data" in our AQA stuff if we're not validating that it's actually bad. Also not sure if it's worth requiring a change, but wanted to think about it

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by athomas-git — DISMISSED** [review]

File: e2e/resources/RWT_Upload_File.csv (hunk 1 above)
Comment (palakpatel311) [reply e2e/resources/RWT_Upload_File.csv:4]: I wouldn't consider it as band data. If a record doesn't have url in the file, we remove url from a  listing. Also, this code doesnt even deal with what is supplied in the file.

**Review by andlar — DISMISSED** [review]

**Review by athomas-git — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

File: e2e/components/listing/edit/listingEdit.po.js [hunk 2]
```diff
@@ -63,10 +63,10 @@ class ListingEditComponent {
         this.editcertifiedProduct.click();
         if (cures) {
             //click on Edit for on the criteria
-            $('//*[@id="criteria_' + editCriteriaId + '_details_header_cures"]/span[3]/span/button/i').click();
+            $('//*[@id="criteria_' + editCriteriaId + '_details_header_cures"]/div/div[2]/span/button/i').click();
```
Comment (andlar) [inline e2e/components/listing/edit/listingEdit.po.js]: Would this work as something like:
```
$('#criteria_' + editCriteriaId + '_details_header_cures).$(button).click();
```
That way, the next time the structure of the page changes, as long as there's only one button under the criteria, it'll find it? If there are more than one button then maybe that `$(button)` could be `#(button=Details)` or something similar.

File: e2e/components/upload/upload-rwt/upload-rwt.po.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+const uploadElements = {
+    root: 'chpl-upload-real-world-testing',
+    title: '.panel-title',
+    chooseUploadRwt: '//*[@id="ngf-label-upload-button-rwt"]/input[@id="ngf-upload-button-rwt"]',
+    uploadButton: '.btn.btn-ai-success',
+    fileUploadText: '//*[@id="main-content"]/div/ui-view/chpl-upload/div/div/chpl-upload-real-world-testing/div/div[2]/div',
```
Comment (andlar) [inline e2e/components/upload/upload-rwt/upload-rwt.po.js]: Why are we going back out to the `main-content` id? the `elements.root` value is in this locator, so why not go to that element followed by the train of `div`s?

**Review by kekey1 — DISMISSED** [review]

File: e2e/components/listing/edit/listingEdit.po.js (hunk 2 above)
Comment (palakpatel311) [reply e2e/components/listing/edit/listingEdit.po.js]: Used $$(button)[1] as button with text did not work

File: e2e/components/upload/upload-rwt/upload-rwt.po.js (hunk 3 above)
Comment (palakpatel311) [reply e2e/components/upload/upload-rwt/upload-rwt.po.js]: Fixed this

**Review by andlar — APPROVED** [review]

**Review by athomas-git — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Pull request merged**